ut.h

```diff
@@ -58,6 +58,12 @@ struct sip_msg;
 /* the amount of decimals to be displayed for "float" and "double" values */
 #define FLOATING_POINT_PRECISION 8
 
+/* zero-string redaction wrapper */
+#ifdef REDACT_LOG
+	#define ZSRW(_c) ((_c)?("****"):"")
+#else
+	#define ZSRW(_c) ((_c)?(_c):"")
```

Comment:
How about `#define ZSRW(_c) ZSW(_c)` to eliminate code duplication? This also requires reordering the macros.